src/configure

```diff
@@ -1090,22 +1090,26 @@ fi
 echo "OPENFSTINC = $FSTROOT/include" >> kaldi.mk
 if $static_fst ; then
   OPENFSTLIBS="$FSTROOT/lib/libfst.a"
+  OPENFSTSCRIPTLIBS="$FSTROOT/lib/libfstscript.a"
 else
   if [ "`uname`" == "Darwin"  ]; then
     OPENFSTLIBS="$FSTROOT/lib/libfst.dylib"
     OPENFSTLDFLAGS="-Wl,-rpath -Wl,${FSTROOT}/lib"
+    OPENFSTSCRIPTLIBS="$FSTROOT/lib/libfstscript.dylib"
```

Comment:
oh.  I didn't realize that we have to link an extra library in order to use the FstScript stuff.
Doing that would require us to bump up the configure version, forcing everyone to re-run configure.
Since this change does not produce any measurable benefit for the user, let's revert that part of the PR.